tests/test_routes.py

```diff
@@ -12,6 +12,7 @@
     "DATABASE_URI", "postgres://postgres:postgres@localhost:5432/testdb"
 )
 BASE_URL = "/wishlists"
+ITEM_URL = BASE_URL + "/1/items"
```

Comment:
should we assume it is always `1` or take this value from whatever the factory generates?

Reply:
hmmm... good point.  does `nose` always wipe the database clean before each run?

Reply:
It will wipe the db each time, this is fine but I can change it.